### PR TITLE
Ele 3919 docs fixes 

### PR DESCRIPTION
**Pull request by NoyaArie**

### CODE REVIEW

**Review by ellakz — APPROVED** [review]

File: docs/features/anomaly-detection/metrics.mdx [hunk 1]
```diff
@@ -84,10 +84,10 @@ models:
             type: row_count
           - name: filtered_row_count
             type: row_count
-            where_expression: country = 'USA'
           - name: null_count
             type: null_count
             columns: [hello, world]
+        where_expression: country = 'USA'
         cloud_monitored=True
```
Comment (joostboon) [inline docs/features/anomaly-detection/metrics.mdx:91]: Also this seems invalid YAML

File: docs/features/anomaly-detection/metrics.mdx [hunk 2]
```diff
@@ -84,10 +84,10 @@ models:
             type: row_count
           - name: filtered_row_count
             type: row_count
-            where_expression: country = 'USA'
           - name: null_count
             type: null_count
             columns: [hello, world]
+        where_expression: country = 'USA'
```
Comment (NoyaArie) [inline docs/features/anomaly-detection/metrics.mdx:90]:
```suggestion
        where_expression: "country = 'USA'"
```
@joostboon wdyt?

Comment (joostboon) [reply docs/features/anomaly-detection/metrics.mdx:86]: 👌

**Pull request merged**